### DISCUSSION TITLE
Multi User Information

### DISCUSSION THREAD

**Discussion by ph0** [navidrome/navidrome#2306] [General]
I noticed the "Top rated" in the Demo. Does the multi-user work so everyone is seeing the top rated and in the menu I can see all of the different users everyone can see recently added tracks on the main page or I guess I am just curious about to what extent is this multi-user or is it just they get their own login and their own playlist and there is zero interaction from user to user?

**Comment by deluan** [comment]
Sorry for overlooking this, but support for Navidrome should be discussed in the [project's repository](https://github.com/navidrome/navidrome/discussions)

Anyway, to answer your question, each user has their own playcounts, playlists, favorites and ratings. So any list in the UI that depends on these values are personalized by user, everything else is common for all users. This makes the Album's lists behave like that:
- All (common)
- Random (common)
- Favourites (personal)
- Top Rated (personal)
- Recently Added (common)
- Recently Played (personal)
- Most Played (personal)

Playlists are private (only the owner can see them), but they can be made "public", meaning all users can see them, but not modify. Admins can see and modify any playlist.

Hope this answers your question.
